client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -125,11 +125,23 @@ const DataFilesToolbar = ({ scheme, api }) => {
     });
   };
 
+  const permissionParams = { files: selectedFiles, scheme, api };
+  const canDownload = getFilePermissions('download', permissionParams);
```

Comment:
Instead of moving these up here, could we keep them where they were before and instead move the `const download` function below? That way the `canDownload` sticks with the other `can` functions